packages/@glimmer/integration-tests/lib/suites/components.ts

```diff
@@ -555,6 +555,27 @@ export class BasicComponents extends RenderTest {
     this.assertStableRerender();
   }
 
+  @test({
+    kind: 'glimmer',
+  })
+  'invoking dynamic component (path) via angle brackets with named block'() {
+    class TestHarness extends EmberishGlimmerComponent {
+      public Foo: any;
+    }
+    this.registerComponent(
+      'Glimmer',
+      'TestHarness',
+      '<this.Foo><:bar>Stuff!</:bar></this.Foo>',
```

Comment:
should it be `this.args.Foo` or `@Foo` ?

Reply:
No, the test is correct. It could use `@Foo` if we wanted, but the specific point of the test is that using a path works.